### PR TITLE
Avhrr

### PR DESCRIPTION
**Pull request by rzuritamilla**

### CODE REVIEW

**Review by romulogoncalves — COMMENTED** [review]
Rename Contunous to Contiguous.

File: avhrr/README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ After unzipping the files donwloaded from EE to a single folder we run the [proc
 This pre-processing is needed because these files will be used together with the spring indices produced using [Google Earth Engine](https://earthengine.google.com/) 
 
 The coordinates of the bounding box needed to run gdalwarp were obtained using QGIS.  
-The shapefilefile statesContuguos can be found [here](/.../...).  
+The shapefilefile statesContuguos can be found [here](statesContuguos.zip). The typo in the filename is mantained for "historical reasons" ;-).
```
Comment (romulogoncalves) [inline avhrr/README.md]: Remove this and rename the file.

**Review by romulogoncalves — APPROVED** [review]

**Comment by romulogoncalves** [comment]
Do not forget to merge with the master since Modis was just merged with it.

File: avhrr/README.md (hunk 1 above)
Comment (rzuritamilla) [reply avhrr/README.md]: solved by https://github.com/phenology/hsr-phenological-modelling/pull/15/commits/ad80cb5f2f954a9c9984102b0d3ee529295c8540

**Comment by rzuritamilla** [comment]
solved by 5d8c1d1

**Comment by romulogoncalves** [comment]
ready for merge

**Pull request merged**